src/components/PopupwithDelete.js

```diff
@@ -0,0 +1,20 @@
+import Popup from "./Popup";
+
+export default class PopupwithDelete extends Popup {
```

Comment:
`PopupwithDelete `

Every variable in the project should be in `camelCase` 

But according to the task, you need to name it `PopupWithConfirmation` https://snipboard.io/fQTnHk.jpg